patchwork-enum-hacks/src/main/java/com/patchworkmc/impl/enumhacks/PatchworkSpawnRestrictionLocation.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package com.patchworkmc.impl.enumhacks;
+
+import net.minecraftforge.common.util.TriPredicate;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.SpawnRestriction;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.ViewableWorld;
+
+public interface PatchworkSpawnRestrictionLocation extends HackableEnum<SpawnRestriction.Location> {

```

Comment:
IMO the HackableEnum part should just be in the mixin, unless our code relies on this interface implementing hackableenum.